Automattic/jetpack
# Admin Page: Add Likes module settings to Admin Page

#### Changes proposed in this Pull Request:

* Adds settings for the **LIkes** module under the **Engagement** tab.

#### Testing instructions:
1. Get to the **Likes** Card under the **Engagement** tab, change the only value
1. Save
1. Refresh the page and check the change was persisted
